src/Native/System.Native/pal_process.cpp

```diff
@@ -3,16 +3,55 @@
 
 #include "../config.h"
 #include "pal_process.h"
-#include <sys/resource.h>
+
 #include <assert.h>
 #include <errno.h>
+#include <signal.h>
 #include <stdlib.h>
+#include <sys/resource.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <syslog.h>
 #include <unistd.h>
-#include <signal.h>
 
 // Validate that our Signals enum values are correct for the platform
-static_assert((int)Signals::PAL_None == 0, "");
-static_assert((int)Signals::PAL_SIGKILL == SIGKILL, "");
+static_assert(static_cast<int>(Signals::PAL_None) == 0, "");
+static_assert(static_cast<int>(Signals::PAL_SIGKILL) == SIGKILL, "");
+
+// Validate that our WaitPidOptions enum values are correct for the platform
+static_assert(static_cast<int>(WaitPidOptions::PAL_None)        == 0, "");
+static_assert(static_cast<int>(WaitPidOptions::PAL_WNOHANG)     == WNOHANG, "");
+static_assert(static_cast<int>(WaitPidOptions::PAL_WUNTRACED)   == WUNTRACED, "");
+
+// Validate that our SysLogPriority values are correct for the platform
```

Comment:
Can you add comments about whether we're just lucky so far or the numeric values are specified by POSIX? I've been making an effort to do that with these blocks.